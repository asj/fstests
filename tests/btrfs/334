#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2025 Oracle.  All Rights Reserved.
#
# FS QA Test 334
#
# Verify sysfs knob input syntax for allocation/data/chunk_size
#
. ./common/preamble
_begin_fstest auto quick

. ./common/sysfs
. ./common/filter

_require_test
_require_fs_sysfs_attr $TEST_DEV allocation/data/chunk_size

_verify_sysfs_syntax $TEST_DEV allocation/data/chunk_size 256m

status=0
exit
