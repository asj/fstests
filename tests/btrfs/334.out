QA output created by 334
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
